### DISCUSSION TITLE
How long is GitHub Education valid for?

### DISCUSSION THREAD

**Discussion by flexprojecthub** [community/community#147859] [GitHub Education] [answered]
### Hi everyone,

How long is GitHub education valid for, and how long can GitHub education be fully utilized? How many times can we apply it?

**Comment by Ravindulakmina** [comment, marked as answer]
GitHub Education (Student Developer Pack) is valid for 1 year from the approval date. After 1 year, you must re-verify your student status to continue using it. You can use GitHub Education Benefits as long as you are a student and successfully update (reapply) your status annually.

**Reply by flexprojecthub** [reply]
Thank you very much for sharing your experience.

**Comment by davevad93** [comment]
Hi @flexprojecthub ,

Actually, `Student Developer Pack` benefits last for 2 years as indicated in the [Student-Developer-Pack-Current-Partners-FAQ](https://github.com/github-education-resources/Student-Developer-Pack-Current-Partners-FAQ):

> First confirm that your student status is still valid (note that student validation lasts for 2 years, after which you will need to revalidate your student status).

You can check the exact expiration date on your [billing summary](https://github.com/settings/billing/summary) in the "**Your recurring charges & discounts**" section.

**Reply by flexprojecthub** [reply]
Thanks, I'll look into what you said.

**Reply by davevad93** [reply]
You're welcome @flexprojecthub 👍

**Comment by AZeed18** [comment]
I will no longer be a student within months, can I benefit for the remaining of the 2 years?

**Reply by Ravindulakmina** [reply]
@AZeed18 If your GitHub Education year has expired, you can retake it if you are a student. You cannot retake it if you are not a student.

**Reply by xnnnsets** [reply]
You can try

**Reply by davevad93** [reply]
Hey @AZeed18, it depends on when your benefits expire, you can check it in your [payment information settings](https://github.com/settings/billing/payment_information) under "`Coupon`".

Student benefits last for 2 years from the date of approval, i.e., if you were verified 6 months ago but in a few months you will no longer be a student, you will continue to have access to your benefits until they expire, which would be until 2027.

**Comment by laxmiramangudewar** [comment]
This is valid for 1 year and after that you need to reapply.
